src/api/stats/getAmmPrices.js

```diff
@@ -58,15 +58,17 @@ const marshPools = require('../../data/degens/marshLpPools.json');
 const lavaPools = require('../../data/lavaLpPools.json');
 const popsiclePools = require('../../data/popsicleLpPools.json');
 const comethPools = require('../../data/comethLpPools.json');
+const hfiPools = require('../../data/hfiLpPools.json');
 
 const INIT_DELAY = 60 * 1000;
 const REFRESH_INTERVAL = 10 * 60 * 1000;
 
 // FIXME: if this list grows too big we might hit the ratelimit on initialization everytime
 // Implement in case of emergency -> https://github.com/beefyfinance/beefy-api/issues/103
 const pools = [
+  ...hfiPools,
   ...comethPools,
-  ...popsiclePools,
+  //...popsiclePools,
```

Comment:
@MirthFutures  this is needed?

Reply:
When I ran the api locally, it kept crashing if I didn't comment out popsicle.

Reply:
should be not related with it, works on prod.
but i saw superb also commented it before.
very weird, its just another cake LP